hexa/ui/datagrid.py

```diff
@@ -347,11 +327,6 @@ def format_date(self, date: datetime.datetime):
         if date is None:
             return date
 
```

Comment:
Please don't do this :)

In this case, you are probably breaking `DateColumn` for existing usages. Just change it for `DAGRunGrid`, not everywhere...